dds/monitor/SPMonitorImpl.cpp

```diff
@@ -46,8 +46,15 @@ SPMonitorImpl::report()
     report.host = this->hostname_.c_str();
     report.pid  = this->pid_;
     DDS::DomainParticipantFactory_var pf = TheParticipantFactory;
-    const DomainParticipantFactoryImpl::DPMap& participants =
-      dynamic_cast<DomainParticipantFactoryImpl*>(pf.in())->participants();
+
+    OpenDDS::DCPS::DomainParticipantFactoryImpl* pi = dynamic_cast<DomainParticipantFactoryImpl*>(pf.in());
```

Comment:
Please make this a separate PR, it's not related to the other changes.

Reply:
Will do.